docs/4.4/enterprise/ssh-rbac.md

```diff
@@ -271,25 +261,57 @@ spec:
 ```
 
 
-## RBAC for Sessions
+## Teleport Resources
+
+RBACs lets teams limit what resources are available to teleport users. This can be
+helpful if you don't want regular users editing SSO `auth_connector` or creating and
+editing new roles `role`.
 
-As shown in the role example above, a Teleport administrator can restrict
-access to user sessions using the following rule:
+As list of all options is below.
 
 ```yaml
-rules:
-  - resources: [session]
-    verbs: [list, read]
+  allow:
+    rules:
+    # Can create and list Teleport Roles.
```

Comment:
This description comments implies, the following setting can only create and list (same thing with others)